docs/guide/admin_panel.md

```diff
@@ -131,17 +131,45 @@ Further manipulation of trash requires the `content/restore` Policy to restore i
 
 #### Restrict editing to part of the tree
 
-If you want to let the User create or edit content, but only in one part of the content tree, you need to use Limitations.
-Three Limitations that could be used here are `Section` Limitation, `Location` Limitation and `Subtree of Location` Limitation.
+If you want to let the User create or edit content, but only in one part of the content tree, you need to use Limitations. Three Limitations that could be used here are `Section` Limitation, `Location` Limitation and `Subtree of Location` Limitation.
 
-Let's assume you have two Folders under your Home: Blog and Articles.
-You can let a User create content for the blogs, but not in Articles by adding a `Subtree of Location` Limitation on the Blog Content item.
-This will allow the User to publish content anywhere under this Location in the structure.
+##### Section Limitation
 
-A `Section` Limitation can be used similarly, but a Section does not have to belong to the same Subtree of Location in the content structure, any Locations can be assigned to it.
+Let's assume you have two Folders under your Home: Blog and Articles. You can let a User create content for the blogs, but not in Articles by adding a `Section` Limitation the Blog Content item. This will allow the User to publish content anywhere under this Location in the structure. Section does not have to belong to the same Subtree of Location in the content structure, any Locations can be assigned to it.
```

Comment:
```suggestion
Let's assume you have two Folders under your Home: Blog and Articles.
You can let a User create content for the blogs, but not in Articles by adding a `Section` Limitation the Blog Content item.
This will allow the User to publish content anywhere under this Location in the structure.
Section does not have to belong to the same Subtree of Location in the content structure, any Locations can be assigned to it.
```